core/src/smartcontracts/isi/permissions/combinators.rs

```diff
@@ -47,7 +47,7 @@ impl<V: IsAllowed<Operation = Instruction>> IsAllowed for CheckNested<V> {
                     if_box
                         .otherwise
                         .as_ref()
-                        .map_or(ValidatorVerdict::Skip, |otherwise| {
+                        .map_or(ValidatorVerdict::Allow, |otherwise| {
```

Comment:
can you explain this?

Reply:
sure.
Suppose we have only `then` branch and it returns `Allow`, `else` branch will always return `Skip`. 
Then we select least permissive from `Allow` and `Skip` which is `Skip`, so there is no way to `If` instruction to be valid.